examples/fscloud/variables.tf

```diff
@@ -108,3 +108,9 @@ variable "backup_encryption_key_crn" {
   default     = null
   # Validation happens in the root module
 }
+
+variable "enable_elser_model" {
+  type        = bool
+  description = "Set it to true to install and start the Elastic's Natural Language Processing model. [Learn more](https://cloud.ibm.com/docs/databases-for-elasticsearch?topic=databases-for-elasticsearch-elser-embeddings-elasticsearch)"
```

Comment:
would be `install` instead of `start` better option?

Reply:
Both `install` and `start` are there. And actually the code is doing both installing and starting..